foreman/data_refinery_foreman/surveyor/array_express.py

```diff
@@ -16,68 +17,111 @@
 logging.basicConfig(level=logging.INFO)
 logger = logging.getLogger(__name__)
 
+EXPERIMENTS_URL = "https://www.ebi.ac.uk/arrayexpress/json/v3/experiments/"
```

Comment:
I don't think it would because if that URL ever changes then I think there will be additional changes that need to happen to work with that URL. We can also just change this in the code instead of in the `settings.py` file if we need to.